docs/source/index.rst

```diff
@@ -136,6 +136,7 @@ Welcome to AiiDA's documentation!
 .. toctree::
    :maxdepth: 2
    :caption: Internal architecture
+   :hidden:
```

Comment:
This change is unrelated.

Reply:
True, but should be done at some point. Want me to remove it from this one?

Reply:
It's fine.

Reply:
All done